addons/l10n_id_efaktur/views/res_partner_views.xml

```diff
@@ -7,7 +7,6 @@
             <field name="inherit_id" ref="base.view_partner_form"/>
             <field name="arch" type="xml">
                 <field name="vat" position="after">
-                    <field name="country_code" invisible="1"/>
```

Comment:
this kind of cleanup has nothing to do with your task.

Reply:
See https://github.com/odoo/odoo/pull/61118#discussion_r597727076